src/sections/Facet/Facet.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Field from "../../components/Field/FieldContainer";
+import Checkbox from "styled-components"
```

Comment:
![Screen Shot 2020-04-28 at 14 00 27](https://user-images.githubusercontent.com/29271960/80484534-2a6aef80-8958-11ea-984a-6e7f2c591447.png)
I received an import error of the "styled-components", maybe let's clean up the unnecessary imports? :)

Reply:
```suggestion
import React from "react"
import specificationFacetConfig from "../../configs/specificationFacetsConfig";
import "./Facet.css"
import PropTypes from "prop-types"
```
I think that are all the imports that we need :)